packages/account-management/src/frontend/AccountManagement/TabNav.tsx

```diff
@@ -0,0 +1,108 @@
+// Converted from flow to TSX element
```

Comment:
My suggestion as we convert weco components and introduce them to the codebase would be to keep them in a seperate folder for maintainability for tracking which components we've had to do this with?

Reply:
Good idea. Is the `frontend/Shared` folder a good place for it to live? It looks like `frontend/Shared/CheckBoxLabel.tsx` was also adapted from a WeCo component, based on the comments there.

Reply:
Yeah I'd suggest maybe both of them go into a subfolder of shared.... or both out to their own folder not nested in shared. I don't think it really matters as long as we can keep good track of them and it's clear to weco what we've done.